src/TodoList.jsx

```diff
@@ -0,0 +1,22 @@
+
+const todoList = [
+  {id: 1, title: 'Read material for the lesson'},
+  {id: 2, title: 'Do textbook exercises'},
+  {id: 3, title: 'Complete the assignment'}
+];
+
+const TodoList = () => {
+  return (
+    <ul>
+      {todoList.map((item) => {
+        return(
```

Comment:
It looks like you may be missing a space in your return statement here. Not a big deal of course, but I thought I'd point it out in case you missed it!